spec/index.html

```diff
@@ -5702,14 +5702,19 @@ <h5>IN</h5>
 <span class="return">boolean</span>  rdfTerm <span class="operator">IN</span> (<span class="expression">expression</span>, <span class="expression">...</span>)
             </pre>
             <p>The <code>IN</code> operator tests whether the RDF term on the left-hand side is found
-              in the values of list of expressions on the right-hand side. The test is done with "="
+              in the list of values of the expressions on the right-hand side. The test is done with "="
               operator, which tests for the same value, as determined by the 
               <a href="#OperatorMapping">operator mapping</a>.</p>
             <p>A list of zero terms on the right-hand side is legal.</p>
             <p>Errors in comparisons cause the <code>IN</code> expression to raise an error if the
               RDF term being tested is not found elsewhere in the list of terms.</p>
             <p>The <code>IN</code> operator is equivalent to the SPARQL expression:</p>
-            <pre>(lhs = expression1) || (lhs = expression2) || ...</pre>
+            <pre>(rdfterm = expression1) || (rdfterm = expression2) || ...</pre>
```

Comment:
```suggestion
            <pre><code>(rdfterm = expression1) || (rdfterm = expression2) || ...</code></pre>
```

Reply:
`<pre>` includes styling so the `<code>` is unnecessary. 

Elsewhere in the document is it `<pre class=` but I couldn't find an appropriate class (maybe that's why there isn't on in the original).

There is only one `<pre><code>` elsewhere although it is a special note.

There is a lot of styling to be sorted out sometime.

Reply:
OK. I wasn't looking at the preview; just reviewing the HTML.